diff --git a/lib/esm/ModuleChunkLoadingRuntimeModule.js b/lib/esm/ModuleChunkLoadingRuntimeModule.js
index 4a846a7e4ef1c454379b1983c37ac25baa93766e..a16aa2c88ec07de810198e73abdbf21fd32f8980 100644
--- a/lib/esm/ModuleChunkLoadingRuntimeModule.js
+++ b/lib/esm/ModuleChunkLoadingRuntimeModule.js
@@ -193,7 +193,7 @@ class ModuleChunkLoadingRuntimeModule extends RuntimeModule {
 													: `if(${hasJsMatcher("chunkId")}) {`,
 												Template.indent([
 													"// setup Promise in chunk cache",
-													`var promise = ${importFunctionName}(${JSON.stringify(
+													`var promise = ${importFunctionName}(/* webpackInclude: /\.js$/ */ ${JSON.stringify(
 														rootOutputDir
 													)} + ${
 														RuntimeGlobals.getChunkScriptFilename
